api/tests/integration/infrastructure/repositories/account-recovery-demand-repository_test.js

```diff
@@ -47,38 +50,35 @@ describe('Integration | Infrastructure | Repository | account-recovery-demand-re
       });
 
       context('when demand is unused yet', () => {
```

Comment:
**chipotage**
`context('when demand is not used yet', () => {`

Reply:
The text seems to be passive voice. To make the text clearer and easier to read, you can often use the active voice.

Reply:
when demand is not yet used (et tu peux ignorer les passive voice message de l'IDE je pense ;) )